torch/utils/_config_module.py

```diff
@@ -278,6 +289,10 @@ def __getattr__(self, name: str) -> Any:
             if config.hide:
                 raise AttributeError(f"{self.__name__}.{name} does not exist")
 
+            alias_val = self._get_alias_val(name)
```

Comment:
nit - Just pass the config into the function? No need to go fetch it when we already have it here?

Reply:
i dont understand?